routes/booking.rb

```diff
@@ -0,0 +1,18 @@
+class App < Sinatra::Base
+  delegate  :department_title, :arrival_title, :route_id,
+            :first_class_available_seats, :second_class_available_seats,
+            :first_class_price, :second_class_price,
+            :max_first_class_tickets_count, :max_second_class_tickets_count,
```

Comment:
That's a bad way of using `delegate`. You should use solely `@booking` variable inside of the view since it's a presenter & leverage delegation inside of presenter's definition